sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clienttelemetry/CosmosMetricNames.java

```diff
@@ -0,0 +1,147 @@
+            public final static String Available = "rntbd.channels.available.count";
+
+            // Snapshot of the number of closed channels for this endpoint (FunctionCounter)
+            public final static String Closed = "rntbd.channels.closed.count";
+        }
+
+        public final static class Requests {
+            // Latency of RNTBD requests for this endpoint (Timer)
+            public final static String Latency = "rntbd.requests.latency";
+
+            // Latency of failed RNTBD requests for this endpoint (Timer)
+            public final static String FailedRequestLatency = "rntbd.requests.failed.latency";
+
+            // Latency of successful RNTBD requests for this endpoint (Timer)
+            public final static String SuccessRequestLatency = "rntbd.requests.successful.latency";
+
+            // Snapshot of number of concurrent RNTBD requests for this endpoint (Gauge)
+            public final static String Concurrent = "rntbd.requests.concurrent.count";
+
+            // Snapshot of number of queued RNTBD requests for this endpoint (Gauge)
+            public final static String Queued = "rntbd.requests.queued.count";
+
+            // Size of the request payload (DistributionSummary)
+            public final static String RequestPayloadSize = "rntbd.req.reqSize";
+
+            // Size of the response payload (DistributionSummary)
+            public final static String ResponsePayloadSize = "rntbd.req.rspSize";
+        }
+
+        public final static class Endpoints {
+            // Snapshot of the number of endpoints (Gauge)
+            public final static String Count = "rntbd.endpoints.count";
+
+            // Snapshot of the number of evicted/closed endpoints (FunctionCounter)
+            public final static String EvictedCount = "rntbd.endpoints.evicted";
+        }
+    }
+
+    public final static class Operation {
+        // Number of operation calls (Counter)
+        public final static String Calls = "op.calls";
+
+        // Total latency (across requests including retries) of the operation (Timer)
+        public final static String Latency = "op.latency";
+
+        // Actual item count - relevant for non-point-operations - indicating the actual number of
+        // docs returned in the response (DistributionSummary)
+        public final static String ActualItemCount = "op.actualItemCount";
+
+        // Max. item count - relevant for non-point-operations - indicating the requested max. number of
+        // docs returned in a single response (DistributionSummary)
+        public final static String MaxItemCount = "op.maxItemCount";
+
+        // Request charge for the operation (DistributionSummary)
+        public final static String RequestCharge = "op.RUs";
+
+        // Number of regions contacted for processing the operation (DistributionSummary)
+        public final static String RegionsContacted = "op.regionsContacted";
```

Comment:
RegionsContacted belongs to `operationDetails` - I wonder whether can group these based on their category as well

Reply:
Done